scopehal/CMakeLists.txt

```diff
@@ -10,6 +11,11 @@ else()
 	set(LIN_LIBS dl)
 endif()
 
+# adapt for BSD pthreads API
+if((CMAKE_SYSTEM_NAME STREQUAL "FreeBSD") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin"))
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Dst_mtim=st_mtimespec")
```

Comment:
According to https://man.bsd.lv/FreeBSD-12.0/stat.2, freebsd's `struct stat` has the correct `st_mtim`.